lib/get-group-tangle.js

```diff
@@ -42,18 +41,24 @@ module.exports = function GetGroupTangle (server, keystore) {
     }
   }
 
-  return function getGroupTangle (groupId, cb) {
+  return function getGroupTangle (groupId, tangle, cb) {
+    if (typeof tangle === 'function') {
+      cb = tangle
+      tangle = 'group'
+    }
     if (!isGroup(groupId)) return cb(new Error(`get-group-tangle expects valid groupId, got: ${groupId}`))
 
     const info = keystore.group.get(groupId)
     if (!info) return cb(new Error(`get-group-tangle: unknown groupId ${groupId}`))
 
     // if it's in the cache, then get the cached value, then callback
     if (cache.has(groupId)) {
-      return cb(null, {
-        root: info.root,
-        previous: Object.keys(cache.get(groupId).state)
-      })
+      return setTimeout(() => {
+        return cb(null, {
+          root: info.root,
+          previous: Object.keys(cache.get(groupId).state)
+        })
+      }, 50)
```

Comment:
why timeout? Either remove, Or leave a comment about why it's needed please

Reply:
here's the context https://github.com/ssbc/ssb-tribes/pull/88#discussion_r1335065883

maybe i should figure out a better solution idk...

Reply:
added a comment